src/runtime_src/core/pcie/driver/linux/xocl/mgmtpf/mgmt-core.c

```diff
@@ -1077,6 +1077,7 @@ static void xclmgmt_extended_probe(struct xclmgmt_dev *lro)
 
 	if (!(dev_info->flags & XOCL_DSAFLAG_DYNAMIC_IP) &&
 	    !(dev_info->flags & XOCL_DSAFLAG_SMARTN) &&
+	    !(dev_info->flags & XOCL_DSAFLAG_VERSAL_EDGE) &&
```

Comment:
Can you please explain why do we need this flag?

Basically, I see you treat this new platform as regular VCK5000. But use this flag to make it special. What is the reason do we need to treat it specially? Is it because we don't have DMA IP where we can config the MSIX? If so why can't we support it here? Do we plan to have user function on this platform later? Do we need PF mail box between the two functions? 
I am trying to understand if the  XOCL_DSAFLAG_VERSAL_EDGE is just a temporary flag until we have MSI-X support. Or something will be fundamentally different between VCK5000 and this platform?

Reply:
yes this flag is added due to no DMA IP where we configure MSIX, this is custom platform which customer has created they are going to enable DMA in next update of platform.
We are planning to use user functions also in this platform but no communication is required, as of now they are using userpf mainly to get aie errors back to host